content/md/en/docs/tutorials/connect-relay-and-parachains/open-message-passing-channels.md

```diff
@@ -121,9 +125,8 @@ To prepare the encoded call to open a channel:
    
    ![Copy the encoded call data](/media/images/docs/tutorials/parachains/hrmp-encoded-call.png)
    
-   You'll need this information to craft the XCM message.
-   The following is an example of encoded call data in Rococo:
-   0x3c00e90300000800000000001000
+   You'll need this information to construct the XCM message.
```

Comment:
"XCM message" is an oxymoron as the M in XCM already stands for message.

Reply:
good point